packages/shylu/shylu_node/basker/cmake/Dependencies.cmake

```diff
@@ -2,7 +2,8 @@ SET(LIB_REQUIRED_DEP_PACKAGES Kokkos TrilinosSS)
 SET(LIB_OPTIONAL_DEP_PACKAGES)
```

Comment:
On line 1 (won't let me highlight) add Teuchos as a required package to accommodate header use like in packages/shylu/shylu_node/basker/src/shylubasker_matrix_def.hpp